examples/embed/example_vertexai.py

```diff
@@ -0,0 +1,24 @@
+from unstructured.documents.elements import Text
+from unstructured.embed.vertexai import VertexAIEmbeddingConfig, VertextAIEmbeddingEncoder
```

Comment:
misspell encoder - Vertext...

Reply:
[working example in examples with comments updated](https://github.com/Unstructured-IO/unstructured/pull/2693/commits/5b278bf93fe0fd08eb0a97ed66485f049c960d6c)